### DISCUSSION TITLE
Can't change native tab switching keybinds

### DISCUSSION THREAD

**Discussion by jsardev** [ghostty-org/ghostty#3718] [Issue Triage] [answered]
I'd like to have tab left/right switching via `cmd+[` and `cmd+]` and tab move action on `cmd+shift+[` and `cmd+shift+]`. The first works fine, but the second doesn't:

```
keybind = cmd+right_bracket=next_tab
keybind = cmd+left_bracket=previous_tab
keybind = cmd+shift+right_bracket=move_tab:+1
keybind = cmd+shift+left_bracket=move_tab:-1
```

**Comment by eliasnaur** [comment]
I'm on macOS and I also can't get `move_tab` to work with

```
keybind = shift+super+left_bracket=move_tab:-1
keybind = shift+super+right_bracket=move_tab:+1
```

The shortcuts behave as `next_tab` and `previous_tab` instead.

Weirdly, setting

```
keybind = super+left_bracket=move_tab:-1
keybind = super+right_bracket=move_tab:+1
```

*does* work.

In addition, `move_tab` is documented to "If the new position is out of bounds, it wraps around cyclically within the tab range.", but I don't see this behavior; tabs move all the way to the (left or right) end and stay there.

**Reply by jsardev** [reply]
@eliasnaur I believe that `shift+super+left_bracket` is the native keybind so even if you remove it it will work that way. I think it may be something to do with a conflict with [native keybinds](https://github.com/ghostty-org/ghostty/issues/414).

**Reply by rhodes-b** [reply]
@jsardev you are correct if is a system keybind ghostty cant remap them currently
#3187

**Comment by mitchellh** [comment, marked as answer]
This is going to be resolved with Ghostty 1.2 since named keys like `left_bracket` now map to _physical_ keys and not translated ones.
